### PR TITLE
Env variables support

### PR DESCRIPTION
**Pull request by seiyanuta**

### CODE REVIEW

File: app/controllers/baseos_controller.rb [hunk 1]
```diff
@@ -42,6 +42,12 @@ def image
     handle_range_header(image_params[:deployment_id], deployment.image)
   end
 
+  def envvars
+    vars = Envvar.where(device: @device).select("name", "value").all
```
Comment (sh19910711) [inline app/controllers/baseos_controller.rb]: We can use `@device.envvars`?

Comment (seiyanuta) [inline app/controllers/baseos_controller.rb]: 👍

Comment (seiyanuta) [inline app/controllers/baseos_controller.rb]: fixed

**Comment by sh19910711** [comment]
Seems good :+1: Thanks.

**Pull request merged**